content/en/opentelemetry/integrations/log_collection.md

```diff
@@ -8,6 +8,10 @@ further_reading:
   text: "Setting Up the OpenTelemetry Collector"
 ---
 
+<div class="alert alert-info">
+The Datadog Agent logs pipeline will be enabled by default in the Datadog Exporter in Collector version 0.108.0. This may cause a breaking change if <a href="https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L456-L462">`logs::dump_payloads`</a> is in use while upgrading, since this option is invalid when the Datadog Agent logs pipeline is enabled. To avoid this issue, remove the `logs::dump_payloads` config option or temporarily disable the `exporter.datadogexporter.UseLogsAgentExporter` feature gate.
```

Comment:
```suggestion
The Datadog Agent logs pipeline is enabled by default in the Datadog Exporter in Collector version 0.108.0. This may cause a breaking change if <a href="https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L456-L462">`logs::dump_payloads`</a> is in use while upgrading, since this option is invalid when the Datadog Agent logs pipeline is enabled. To avoid this issue, remove the `logs::dump_payloads` config option or temporarily disable the `exporter.datadogexporter.UseLogsAgentExporter` feature gate.
```